gtsfm/frontend/verifier/ransac.py

```diff
@@ -23,9 +23,7 @@
 from gtsfm.common.keypoints import Keypoints
 from gtsfm.frontend.verifier.verifier_base import VerifierBase
 
-
-NORMALIZED_COORD_RANSAC_THRESH = 0.001  # TODO: hyperparameter to tune
-PIXEL_COORD_RANSAC_THRESH = 0.5  # TODO: hyperparameter to tune
+PIXEL_COORD_RANSAC_THRESH = 4  # TODO: hyperparameter to tune
```

Comment:
any reason for increasing the threshold?

Reply:
@ayushbaid so I noticed that COLMAP uses 3 or 4 px as its threshold. I think this should go in a global config, since it's such an important hyperparameter. COLMAP seems to use the same pixel threshold when fitting both E, F, and H

Reply:
Makes sense.

Reply:
Maybe create an issue for this?

Reply:
sure -- will do